src/maxdiffusion/models/train.py

```diff
@@ -451,16 +496,14 @@ def compute_loss(unet_params):
         step_num = step + 1
         samples_count = total_train_batch_size * step_num
         if step != 0 and samples_count % config.checkpoint_every == 0:
-           if config.eval_at_checkpoint:
-              eval_at_checkpoint(config,
-                   f"{str(step * total_train_batch_size)}",
-                   unet_state, unet_state_mesh_shardings,
-                   vae_state,
-                   vae_state_mesh_shardings,
-                   pipeline, params, mesh, rng)
-           checkpoint_name = f"{step_num=}-{samples_count=}"
```

Comment:
This change will break the later evaluation 
* extract step_name and samples count info from the name https://github.com/google/maxdiffusion/blob/mlperf_4/src/maxdiffusion/eval.py#L87
Is it possible to keep the `checkpoint_name` format?